cmd/fyne_demo/screens/bindings.go

```diff
@@ -0,0 +1,183 @@
+package screens
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/binding"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+)
+
+// BindingsScreen loads a data bindings example panel for the demo app
+func BindingsScreen() fyne.CanvasObject {
+	// TODO entry
+	// TODO form
```

Comment:
I don't think we have to bind everything - really it's aimed at data display elements isn't it?
form/group/scroller for example surely have no bindable content?

Reply:
Sure. (thinking out loud) binding to scroller could be a way to implement the "scroll to cursor" functionality we talked about

Reply:
Hmm, I'm not sure about this. It seems very strange that an item of data setting scroll position would make sense - scrolling is a user controlled activity (for the most part).
As before I think that data binding is about presenting and adjusting data primarily - to venture into UI state is a very different project and would need appropriate planning over and above the simple data binding concept.